public/index.html

```diff
@@ -16,11 +18,145 @@
     <!-- initialize the SDK after all desired features are loaded -->
     <script defer src="/__/firebase/init.js"></script>
 
-
 </head>
 
 <body>
-    Hello 12
+    
+    <nav class="navbar navbar-expand-lg sticky-top navbar-dark bg-dark">
+        
+            <a class="navbar-brand text-white px-2" style="font-weight: bold;" href="index.html">
+                <img src="image/logo.png" width="30" height="30" alt="" loading="lazy">
+                Work Trolly
+            </a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText"
+                aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon text-white"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarText">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active pr-5 py-1">
+                        <button type="button" class="btn btn-outline-info" id="createNewTaskButton">Create New
+                            Task</button>
+                    </li>
+                    <li class="nav-item pr-5 py-1">
+                        <button type="button" class="btn btn-outline-info">All Task</button>
+                    </li>
+                    <li class="nav-item py-1">
+                        <button class="btn btn-primary" type="button">Login</button>
+                    </li>
+                </ul>
+
+            </div>
+        
+    </nav>
```

Comment:
These are codes from your older commit
Please fix that, and make sure when you are working on the new branch you don't put codes from other branch, it doesn't sound meaningful but git will merge both the files and will make single file using 2 different and finally when merged it will not create any problem.

Reply:
okay i will remove navbar

Reply:
actually i have added navbar code in this branch to test it.
i will remove these code.